packages/authhero/src/utils/crypto.ts

```
@@ -0,0 +1,37 @@
+  }
+}
+
+export async function computeCodeChallenge(
+  codeVerifier: string,
+  method: "plain" | "S256",
+): Promise<string> {
+  switch (method) {
+    case "plain":
+      return codeVerifier;
+    case "S256":
+      const encodedData = new TextEncoder().encode(codeVerifier);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)